src/js/update-sites.js

```diff
@@ -20,26 +22,29 @@ let vm = new Vue({
     packages: [],
     codes: [],
     sites: [],
-    siteQuery: []
+    siteQuery: '?where={"code.package":"588b90ab36447521c5af51e4"}'
   },
   methods: {
     getSites(siteQuery) {
       let that = this;
-      let endpoint = '/sites';
+      let endpoint = 'sites';
+      console.log(packageBaseURL+endpoint+siteQuery);
 
-      atlasRequest(baseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      atlasRequest(packageBaseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      setTimeout(remDup, 50);
     },
     getCodes() {
       let that = this;
-      let endpoint = '/code';
+      let endpoint = 'code';
+      console.log(packageBaseURL+endpoint);
 
-      atlasRequest(baseURL, endpoint, query = '', method = 'GET', body = null, etag = null).then(data => that.codes = data);
+      atlasRequest(packageBaseURL, endpoint, query = '', method = 'GET', body = null, etag = null).then(data => that.codes = data);
     },
     sendToAtlas() {
-      let siteOriginalPackages = this.sites._items;
+      let siteOriginalPackages = this.sites[0];
```

Comment:
This is what I'm getting back with {{sites}}:

`[ [ { "code": { "profile": "59910dac3644752d498d79da", "core": "59836e68364475026576d2c3", "package": [ "588b90a93644752164c7beae", "588b90ab36447521657fd40f", "588b90ab36447521c5af51e4", "588b90ad364475217a9fcf32", "588b90ad36447521657fd411", "59398ded3644755921511d13" ] }, "created_by": "jafu6031", "statistics": "588f8d113644753e810ef343", "modified_by": "KERE7580", "_deleted": false, "_links": { "self": { "href": "sites/588f8d103644753e838b273b", "title": "site" } }, "sid": "p1c4038dad8c", "type": "express", "status": "launched", "_latest_version": 39, "f5only": false, "db_key": "67414141414142596a343052504a5944424176716d4171583269564454446f70337252486c4a50396231796963736d38344478674a47627a315864626f334c506652386c456d4c536f323469347a76304c773956656457516f4343344556495746706b69366d6e3638597474317a3649455667356a585f62794e702d2d36427777314d62373549412d346271", "path": "webcentral", "pool": "poolb-express", "_updated": "2017-08-14 03:09:57 GMT", "update_group": 11, "settings": { "page_cache_maximum_age": 3600 }, "_version": 39, "import_from_inventory": true, "_created": "2017-01-30 18:59:28 GMT", "_id": "588f8d103644753e838b273b", "_etag": "505e056b83d498914cd5366d8c63ff291996efc6" }, { "code": { "profile": "59910dac3644752d498d79da", "core": "59836e68364475026576d2c3", "package": [ "588b90a036447521fd349bc2", "588b90a036447521c5af51d8", "588b90a13644752166c62225", "588b90a136447521fd349bc4", "588b90a136447521633019af", "588b90a236447521c5af51da", "588b90a23644752166c62227", "588b90a236447521fd349bc6", "588b90ab36447521c5af51e4", "589387203644754f7d3295d3", "58a615fc364475514798f9b7", "58b995c53644755c0c936f00", "58e7f86b3644750838d3a508", "592da2d93644756b06fae298", "592eec973644750954a2f28a", "592eedb136447569d6a52639", "5931c1d236447550e4d39a3e", "59398ded3644755921511d13", "5963d4fb36447508b0b83880", "596e1a65364475781b834847", "5989cf873644755ff4194732", "588b90ab36447521c5af51e4", "589387203644754f7d3295d3", "588b90ab36447521c5af51e4", "589387203644754f7d3295d3", "588b90ab36447521c5af51e4", "58a713c0364475207b295d51", "588b90ab36447521c5af51e4", "589387203644754f7d3295d3", "588b90ab36447521c5af51e4", "58a713c0364475207b295d51", "588b90a036447521fd349bc2", "588b90a736447521fd349bcc" ] }, "created_by": "jafu6031", "statistics": "588fae5436447566b3b45f04", "modified_by": "JEOR0980", "_deleted": false, "_links": { "self": { "href": "sites/588fae543644756cfc235068", "title": "site" } }, "sid": "p15ac50597de", "type": "express", "status": "launched", "_latest_version": 28, "f5only": false, "db_key": "67414141414142596a36355565564666434e5a415479375574563336536b486774794633617169706b344847643169744437487730735f694c596f49443834716d754345783841686b56704c39384864734d564971464a6468626133706d434d63722d446c705132776e434257773867423541644537415578664d46647278356238564b726d354158305964", "path": "cas", "pool": "poolb-express", "_updated": "2017-08-14 21:41:05 GMT", "update_group": 11, "settings": { "page_cache_maximum_age": 3600 }, "_version": 28, "import_from_inventory": true, "_created": "2017-01-30 21:21:24 GMT", "_id": "588fae543644756cfc235068", "_etag": "0b8e4f75b50d0a4c8db608cbdb2890957e790298" }, { "code": { "profile": "59910dac3644752d498d79da", "core": "59836e68364475026576d2c3", "package": [ "588b90ab36447521c5af51e4", "58a615fc364475514798f9b7", "58a713c0364475207b295d51", "588b90ab36447521c5af51e4", "58a713c0364475207b295d51" ] }, "created_by": "osr_atlas_client", "statistics": "589538ac3644757dd2a371fc", "modified_by": "KERE7580", "_deleted": false, "_links": { "self": { "href": "sites/589538ac3644754f0a53334c", "title": "site" } }, "sid": "p158837d3f1e", "type": "express", "status": "launched", "_latest_version": 43, "f5only": false, "db_key": "67414141414142596c5469734d6c3155324c4a645f437359515833375146635a7873513161534f4e75393969416266616d62516c6978784c38556c7179485552586d4357344e5253393770534443304855685567744a535832496a2d75386671585a37736b66446f4354756950634f386268384b3064445639574b542d472d72783278526f2d303052333865", "path": "crowdfunding", "pool": "poolb-express", "_updated": "2017-08-14 03:10:09 GMT", "dates": { "launched": "2017-02-13 23:07:09 GMT", "created": "2017-02-04 02:12:59 GMT" }, "update_group": 0, "settings": { "page_cache_maximum_age": 3600 }, "_version": 43, "import_from_inventory": false, "_created": "2017-02-04 02:12:59 GMT", "_id": "589538ac3644754f0a53334c", "_etag": "8c6c6fef349bfd48eaea55b30b0f60b2ce7d6884" } ] ]`

So the [0] is actually bypassing some random array to get to the nested array of the actual sites, right?